### PR TITLE
Pipeline run tracking

### PR DESCRIPTION
**Pull request by schustmi**
## Describe changes
- It is now possible to set an active project for a local ZenML repository.
- The ZenStore tracks additional information about pipeline runs which are displayed in post-execution.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by stefannica — APPROVED** [review]
I wish I had time to try/test this locally, but I can wait until after it's merged to take it for a spin :)

I don't have any major functionality related comments, and I do understand the choices you had to make to bring this to completion, but I warrant that a re-factoring is in order :)

File: src/zenml/zen_stores/sql_zen_store.py
Comment (schustmi) [reply src/zenml/zen_stores/sql_zen_store.py:142]: Agreed, this was the short-term solution as the entire structure has to change in the future I think and it takes far longer to implement all as separate tables. I'll create tickets for all your requests so we don't lose track after merging this, but I absolutely agree with all of them.

**Pull request merged**
